### ISSUE TITLE
Z axi don't go up 

### ISSUE THREAD

**Issue by Basharaz** [bigtreetech/BIGTREETECH-SKR-CR6#8]
Hi All

I been have problem with my BTT borad. I follow all the instructions but I can't Z axi don't respond to screen command. It's only go down no up at all I tried deffrent things and I bought new btt borad but still same problem.

Troubleshooting is been done.

1- firmware been flashed many times.
2- check all the cables all connected good.
3- use M122 confirm all x, y, Z connect and pass.
4-invert the Z axi + but same problem with deffrent screen.
5- use BTT screen and Creality screen same problem.
6- put back my old Creality borad everything is working without any problems.

Any idea what else can be done?

**Comment by dream111119** [comment]
Hello, please check if the Z-axis motor is installed in reverse.

**Comment by Basharaz** [comment]
Yes I did that many times and try deffrent things but don't work at all.

Thank you
Bashar

**Comment by franklinvv** [comment]
This happens when the current for the Z axis is set too low in Configuration_adv.h, with the Z_CURRENT define directive. You can change it yourself, set it to 1000, and build your own firmware. The current firmware.bin in this repository did not work for me either.

**Comment by Basharaz** [comment]
How to build my own firmware?

**Comment by franklinvv** [comment]
If you're either unable to build or uncomfortable building your own firmware, I would refer you to CR-6 SE community builds here: https://github.com/CR6Community/Marlin/releases

There you can also find builds for the BTT board with either the stock CR-6 SE screen or a BTT TFT, depending on which one you hooked up to your printer. The latest firmware builds are here: https://github.com/CR6Community/Marlin/releases/tag/v2.0.7.3-cr6-community-release-5-beta
